docs/OMS-Agent-for-Linux.md

```diff
@@ -671,12 +671,14 @@ An example section of the `omsagent.conf` configuration file `/etc/opt/microsoft
 </source>
 ```
 # Agent Logs
-The logs for the Operations Management Suite Agent for Linux can be found at: 
-`/var/opt/microsoft/omsagent/<workspace id>/log/`
-The logs for the omsconfig (agent configuration) program can be found at: 
-`/var/opt/microsoft/omsconfig/log/`
-Logs for the OMI and SCX components (which provide performance metrics data) can be found at:
-`/var/opt/omi/log/ and /var/opt/microsoft/scx/log`
+
+Logs for omsagent specifically have a node for the workspace ID.  The  omsconfig directory has one log in an upper level directory, with no log directory named per se.  Here are the directory paths of the four facilities supporting OMS Agent:
+
+1. `/var/opt/microsoft/omsagent/<workspace id>/log/  # (Agent itself; one per workspace)`
```

Comment:
I think it would look cleaner if you put the description before the log path:
OMSAgent logs: `/var/opt/microsoft/omsagent/<workspace id>/log/`

Also, do not mention the possibility of more than one workspace; customers should not be attempting multi-homing yet.